test/system/system_test.go

```diff
@@ -137,6 +137,40 @@ var _ = Describe("Kratix", func() {
 				})
 			})
 
+			PWhen("an existing resource request is updated", func() {
```

Comment:
P

Reply:
it's not passing, since the updates are not completed 🤠